src/Controls/src/Core/Page.cs

```diff
@@ -564,4 +574,53 @@ internal void SetTitleView(View oldTitleView, View newTitleView)
 			_titleView = newTitleView;
 		}
 	}
+
+	// TODO ezhart These Pages could probably all be IView
+	public class PageBusyMessage 
```

Comment:
Nit: all these could be sealed. Also could potentially be records to make them less verbose to declare.

Reply:
Excellent suggestion. Done and done!